src/Files.App/Views/MainPage.xaml.cs

```diff
@@ -82,6 +84,10 @@ public MainPage()
 			UserSettingsService.OnSettingChangedEvent += UserSettingsService_OnSettingChangedEvent;
 
 			DispatcherQueue.TryEnqueue(async () => await LoadSelectedTheme());
+
+			var hotKeyAccelerator = Ioc.Default.GetService<IHotKeyAccelerator>();
```

Comment:
This service is only used in the constructor. It is not necessary to maintain a reference to the instance. I see no reason to make this service required in this piece of code. MainPage initializes the service if it exists, otherwise it's not a problem.

Reply:
So the service is not injected? When is it injected and when not?